install-raspbian.sh

```diff
@@ -69,9 +69,22 @@ apt dist-upgrade -y
 info "### Photobooth needs some software to run."
 apt install -y libapache2-mod-php php-gd gphoto2 unclutter
 
-cd /var/www/
-rm -rf html
-mkdir html
+echo -e "\033[0;33m### Is Photobooth the only website on this system?"
+read -p "### Warning: If typing y, the whole /var/www/html folder will be removed! [y/N] " -n 1 -r deleteHtmlFolder
+echo -e "\033[0m"
+
+if [ "$deleteHtmlFolder" != "${deleteHtmlFolder#[Yy]}" ] ;then
```

Comment:
It's a personal choice. I don't like working with `$REPLY` several times (it gets confusing). In my opinion it's better to work with variables because you can understand the if-case by reading it and without reading the lines above.

Reply:
> you can understand the if-case by reading it and without reading the lines above

That make sense. I like if the code is more readable.